source/Octostache/Templates/TemplateParser.cs

```diff
@@ -3,7 +3,11 @@
 using System.Collections.Specialized;
 using System.Globalization;
 using System.Linq;
+#if NET46
 using System.Runtime.Caching;
+#else
+using Microsoft.Extensions.Caching.Memory;
```

Comment:
Should we use this library in all cases, keeping functionality consistent between target frameworks?
Perhaps we need to do our own cache size limiting.